library/include/hipsparse.h

```diff
@@ -8930,11 +8930,11 @@ typedef enum
     HIPSPARSE_COOMV_ALG        = 1,
     HIPSPARSE_CSRMV_ALG1       = 2,
     HIPSPARSE_CSRMV_ALG2       = 3,
-    HIPSPARSE_SPMV_ALG_DEFAULT = 4,
-    HIPSPARSE_SPMV_COO_ALG1    = 5,
-    HIPSPARSE_SPMV_COO_ALG2    = 6,
-    HIPSPARSE_SPMV_CSR_ALG1    = 7,
-    HIPSPARSE_SPMV_CSR_ALG2    = 8
+    HIPSPARSE_SPMV_ALG_DEFAULT = 0,
+    HIPSPARSE_SPMV_COO_ALG1    = 1,
+    HIPSPARSE_SPMV_CSR_ALG1    = 2,
+    HIPSPARSE_SPMV_CSR_ALG2    = 3,
+    HIPSPARSE_SPMV_COO_ALG2    = 4
```

Comment:
I do not think this is valid?

Reply:
At first I thought it was  typo in cusparse, but they do define the enum using duplicate integers. This apparently is valid and I guess is used to indcate when two enums map to the same algorithm.

Reply:
I believe this just indicates that the enum elements with the same value are equal and can be used interchangeably. For example, when used in a switch statement, only one of the elements with the same value can have a case statement. The other element that doesn't have an explicit case is mapped onto the element with the same value.